lms/static/js/student_account/views/account_settings_fields.js

```diff
@@ -48,6 +79,8 @@
 
         AccountSettingsFieldViews.PasswordFieldView = FieldViews.LinkFieldView.extend({
```

Comment:
This should be added in the same way you have added 3 keys to AccountSettingsFieldViews at the top of the function.